src/libs/utils.js

```diff
@@ -568,13 +568,20 @@ export const getSearchFilterFunctions = () => {
 
       return filter(user => {
         const {
-          displayName, email, roles
+          displayName, email, roles, institution, libraryCards
         } = user;
 
         const matchable = [displayName, email];
         if (!isNil(roles)) {
           matchable.push(...map((r) => r.name)(roles));
         }
+        if (!isNil(institution)) {
+          matchable.push(institution.name);
+        }
+
+        if (!isNil(libraryCards) && !isEmpty(libraryCards)) {
```

Comment:
So there's a discrepancy between the logic here and the logic used to set up data for `permissionsCell`. Here you're checking if the user has a library card at all, whereas the permission cell is checking if the user has a library card where `card.institutionId === institution.id`.

The end result is that the filter will let anyone with a library card show up on the filterTerm `library card`, but the permissions cell will state `None` for any researcher who has a library card that doesn't match their current institution. Since users who are only Researchers are allowed to change institutions, the above scenario can happen.
 
<img width="1585" alt="Screen Shot 2022-09-28 at 9 00 39 AM" src="https://user-images.githubusercontent.com/6664778/192785392-c01ef6f2-a83b-475a-871c-421cfd473ed5.png">

Reply:
ok - so thoughts on rendering library card like, `Library Card (Broad Institute), Library Card (Other Institution), SigningOfficial`?

Reply:
I think some backend changes would need to happen for this. It appears the endpoint is returning library cards, but only sometimes returns the institution alongside the library card. For some library cards, it has: 

```
{
...
institutionId: 120,
institution: { ... name: 'TEST', ... }
}
```

for others 
```
{
...
institutionId: 120,
institution: null
}
```
As such, I made the table consistent (always show library card perm if you have ANY library card) but I did not list all library cards that exist. If we wish to do that, I can investigate the backend work needed & get the table updated accordingly.

Reply:
> As such, I made the table consistent (always show library card perm if you have ANY library card) but I did not list all library cards that exist.

This, in my opinion, is the right choice at this stage. If we want more info in the future, we'll address it then.